exercises/02-module-class/color-generator-2.0.mjs

```diff
@@ -0,0 +1,25 @@
+class ColorsGenerator {
+
+    colors = ['red', 'green', 'blue'];
+    i = 0;
+
+    constructor() {
+    }
+
+    get(n) {
```

Comment:
Not a bug, but during your code reviews in this classroom, if possible please promote [default parameters](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters) usage

I can't complain about your solution, it is smart :D But junior devs tend to solve it in different ways, for example: 
```ts
get(n) {
  const count = n || 1;
}
```
Or:
```ts
get(n) {
  if (!n) {

  }  else ...
}
```

Which could be simpler with default params